modules/eco-configuring-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -3,10 +3,10 @@
 // *nodes/nodes/eco-poison-pill-operator.adoc
 
 :_content-type: PROCEDURE
-[id="configuring-machine-health-check-with-poison-pill_{context}"]
-= Configuring machine health checks to use the Poison Pill Operator 
+[id="configuring-machine-health-check-with-self-node-remediation-operator_{context}"]
```

Comment:
Note that previously poison-pill_ didn't have the `operator` suffix, so unless it was initially wrong I'd expect the same for snr